native/screens/UserProfile.jsx

```diff
@@ -101,33 +111,37 @@ export const UserProfile = ({ route, navigation }) => {
           setReviewStatus={setReviewStatus}
         />
       )}
-      <View>
-        <Text>
+      <View style={styles.nameContainer}>
+      <View style={styles.placeholderImage}></View>
+        <Text style={styles.name}>
           {first_name} {last_name}
         </Text>
         <Text>{role}</Text>
       </View>
 
-      <Text> I AM AN IMAGE</Text>
 
-      <View>
-        <Text>{email}</Text>
-        <Text>PORTFOLIO LINK</Text>
+      <View style={styles.wordContainer}>
+        <Text style={styles.title}>ABOUT</Text>
+        <Text style={styles.words}>{about}</Text>
       </View>
 
-      <View>
-        <Text>{about}</Text>
+      <View style={styles.wordContainer}>
```

Comment:
I noticed we created this container 4 times here. This is a good example of when to use the DRY principle by creating another component that will style and render information passed into it via props. 
That way, instead of:
```js
      <View style={styles.wordContainer}>
         <Text style={styles.title}>ABOUT</Text>
         <Text style={styles.words}>{about}</Text>
      </View>
```
4 times, we can have 1 component that applies all the styles and renders text based on what's passed in
```js
const AboutBox = (text, title) => {
  return (
      <View style={styles.wordContainer}>
          <Text style={styles.title}>{title}</Text>
          <Text style={styles.words}>{text}</Text>
      </View>
 )
}
``` 
Rendered on UserProfile:
```js
<AboutUser title="ABOUT" text="I love coding" />
<AboutUser title="FUN FACT" text="I'm a nerd" />
```
Converting 16 lines of code to just 4

Reply:
Sometimes the function won't work for all of them. For example when you created the View for the `Portfolio` URL with a custom text component that needed an `onPress` prop. 

That's a one-off use case so it's okay to use as you are. However (and dont hate me bcs the idea just came to me), we can still make it work with custom component by just using that component as a container, instead of making it in charge of rendering the info as well. For example:
```js
const const AboutBox = ({ children, text, title }) => {
  return (
      <View style={styles.wordContainer}>
          <Text style={styles.title}>{title}</Text>
          {children}
      </View>
 )
}
```

Which would look like this on UserProfile:
```js
<AboutUser title="about" >
    <Text>Still a nerd</Text>
</AboutUser>

<AboutUser title="about" >
     <Text style={styles.portfolio } onPress={() => Linking.openURL(`${portfolio_link}`)}> Portfolio </Text>
</AboutUser>
```

While it's more verbose, it enables us to be able to use the `AboutUser` component anywhere in the app if we want a box with that same styling

Don't worry if all this feels overwhelming!!! I have a tendency to share what's on my mind all the time but you're doing good! And this kind of exchange of information is super super common in the workplace and may come off like criticism but it's always in the interest of learning!